Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: build
```

Comment:
It seems the `make test` goal is not working, please fix it so that each module can be tested individually.

Reply:
Hi! I have made the corresponding changes to make sure that the test command can be run on each module. We now have the following commands:
 - `make test` - To test all modules
- `make test-provision` - To test provision module
- `make test-install` - To test install module